imaging/ml/ml_codelab/README.md

```diff
@@ -18,33 +18,21 @@ PROJECT_NUMBER=`gcloud projects describe ${PROJECT_ID} | grep projectNumber | se
 COMPUTE_ENGINE_SERVICE_ACCOUNT="${PROJECT_NUMBER}-compute@developer.gserviceaccount.com"
 gcloud projects add-iam-policy-binding ${PROJECT_ID} --member "serviceAccount:${COMPUTE_ENGINE_SERVICE_ACCOUNT}" --role roles/pubsub.admin
 ```
-## Create Datalab instance
+## Create AI Platform Notebooks instance
 
-This following shell command will create a SSH tunnel allowing you to visit the Datalab UI in your web browser. You can run this on a shell in your **local machine** or **[Cloud Shell](https://cloud.google.com/shell/docs/)**.
+Follow the steps listed [here](https://cloud.google.com/ai-platform/notebooks/docs/create-new)
+Create Python 3 instance with deafult n1-standard-4 machine 
 
-```shell
-ZONE="us-central1-c"
-
-gcloud components install datalab
-datalab create mydatalab --machine-type n1-standard-4 --zone ${ZONE}
-```
-
-If it asks you to use datalab-network, you should enter **y** (yes).
-
-When done creating the instance, the shell will print some instruction on how to navigate to the Cloud Datalab UI.
-
-## Set-up Datalab instance
-
-TIP: If you are running on **[Cloud Shell](https://cloud.google.com/shell/docs/)**, you can use "Web Preview" icon (square button on top right of the shell) to connect to Datalab instance at the given port. If you are running on a local machine, you can simply point your browser to the Datalab address that was given above.
+## Set-up AI Platform Notebooks instance
 
-In the Datalab UI, click on the top left to create a new "Notebook".
+In the Jupyter UI, click on the top left to create a new "Notebook" with Python 3.
 
 Then enter the following to import the Git repo containing the ML notebooks/code. This will clone the codelab code to your Datalab environment.
 
 ```ipython
 !git clone https://github.com/GoogleCloudPlatform/healthcare.git
 ```
 
-Then, click on the Google Cloud Datalab logo in the top bar to go back to the Datalab homepage. In the Datalab file UI, navigate to *datalab/healthcare/imaging/ml_codelab*.
+Then, navigate to *datalab/healthcare/imaging/ml_codelab* in a left navigation bar of the JupyterLab UI.
```

Comment:
No *datalab in the beginning
Also the path ends with ml/ml_codelab

Reply:
done